src/main.js

```diff
@@ -2,9 +2,11 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import vueScrollto from 'vue-scrollto'
+import Lightbox from 'vue-easy-lightbox'
```

Comment:
import LightBox just in Gallery view and delete from this file
![image](https://user-images.githubusercontent.com/13311920/53830171-26c12000-3f82-11e9-8b6d-5d16d7bd7897.png)